config/spid-auth.php

```diff
@@ -43,6 +43,7 @@
     'hide_real_idps' => 'production' !== env('APP_ENV'),
     'expose_sp_metadata' => true,
     'expose_idps_json' => false,
+    'only_sp_logout' => true,
```

Comment:
@pdavide quindi non invochiamo mai l'endpoint di logout degli IdP, o sbaglio?

Reply:
Corretto, al momento la cosa non funziona perché siamo su un metadata con diversi acs, quindi facciamo logout solo in locale.